raiden/network/transport/matrix/utils.py

```diff
@@ -347,11 +366,12 @@ def get_reachability_from_matrix(self, user_ids: Iterable[str]) -> AddressReacha
     def _maybe_address_reachability_changed(self, address: Address) -> None:
         # A Raiden node may have multiple Matrix users, this happens when
         # Raiden roams from a Matrix server to another. This loop goes over all
-        # these users and uses the "best" presence. IOW, if there is a single
+        # these users and uses the "best" presence. IOW, if there is at least one
```

Comment:
I just had a discussion with @ulope about the feasability of allowing multiple users of the same address online. This should actually be forbidden per definition because it would imply multiple usage of the same keystore. A raiden node could prevent starting if a user for the same address is found online.

Reply:
The comment change was only to reflect the current code.

Reply:
Then we should ask us if we should change the code. Why should we handle multiple users being online if this should be prevented anyway? 
But on the other hand this could also be implemented in a different PR

Reply:
We should create a follow up issue to discuss this:
- for a canonical client, we can definitely try to forbid multiple logins
- for the peer handling side (such as here), we have to deal with the possibility of multiple logins
- I don't know how _roaming_ user ids behave, when it comes to login/logout. We need to clarify if there are interleaved states, where the old user_id still appears online, while the new user_id already came up.

Reply:
Good point. A user is set to be offline by the server whenever he did not longpoll `sync()` for 30s. This definitely leaves the possibility open to appear online with 2 user ids for the same address.